### PR TITLE
Updating the bgp_adjancies script to work on default VRF without neighbors

### PR DESCRIPTION
**Pull request by KamyarZiabari**

### CODE REVIEW

File: bgp_adjacencies/BGP_Neighbors_Established.py [hunk 1]
```diff
@@ -87,7 +87,9 @@ def check_bgp(self):
         for device, bgp in self.all_bgp_sessions.items():
             # may need to change based on BGP config
             default = bgp['instance']['default']['vrf']['default']
```
Comment (jeaubin) [inline bgp_adjacencies/BGP_Neighbors_Established.py:89]: Let's be safe and also do the same here  default = bgp['instance']['default']['vrf']['default']

this is just ready to blow up too

Comment (tahigash) [reply bgp_adjacencies/BGP_Neighbors_Established.py:89]: or better to change to checking all VRFs instead of only `default`.

**Review by tahigash — CHANGES_REQUESTED** [review]

**Comment by automateyournetwork** [comment]
Yes I have many VRFs can we check them all ? The ISR checks them all I
think

File: bgp_adjacencies/BGP_Neighbors_Established.py (hunk 1 above)
Comment (KamyarZiabari) [reply bgp_adjacencies/BGP_Neighbors_Established.py]: Changed to check all the vrf's

**Review by tahigash — APPROVED** [review]

**Comment by automateyournetwork** [comment]
This PR fixed Nexus 9000 ! Thanks! And it also still works on ISR 4440 (XE)

**Pull request merged**